wurst/abilities/equipment/ShieldAllLevels.wurst

```diff
@@ -1,13 +1,14 @@
 package ShieldAllLevels
```

Comment:
Please split out the Best Practices changes into a separate PR

Reply:
this is from ur bitbucket changes which aren't on github silly